ci/build_container/build_recipes/v8.sh

```diff
@@ -11,19 +11,70 @@ export PATH="$PATH:$PWD/depot_tools"
 
 VERSION=7.5.288.22
 
-fetch v8
-cd v8
-git checkout "$VERSION"
-gclient sync
+cat >.gclient <<\EOF
```

Comment:
Temporary workspace.

NOTE: This PR is more of a stepping stone, and this build script is going to be moved outside of Bazel in a subsequent commit where I'm going to prepare fully fetched archives that can be cached in Bazel, instead of downloading 1.3 GB on each rebuild.